src/interface/others/TalentStatisticBox.js

```diff
@@ -21,6 +21,7 @@ const TalentStatisticBox = ({ talent, icon, label, ...others }) => {
 TalentStatisticBox.propTypes = {
   ...StatisticBox.propTypes,
   talent: PropTypes.number.isRequired,
+  value: PropTypes.node.isRequired, //Ensure this is required
```

Comment:
This is in StatisticBox.propTypes which is expanded here; does the override not work for lint?

Reply:
Nope, only talent is passed through for me

Reply:
As discussed in disc, works as intended for react but IDE's don't interpret the expanded proptypes properly for inspections & autocomplete.... 

Can either leave as-is and rely on standard react proptype behavior (console messages) sacrificing IDE autocomplete and inspector warnings _or_ redo the proptypes completely for talent/trait statistics box to not use any expansion, so that IDE's interpret it all properly. But probably shouldn't do a mix of the two (just adding the one prop)